lib/rubocop/cop/style/indent_heredoc.rb

```diff
@@ -87,14 +120,27 @@ def correct_by_library(node)
           end
         end
 
+        def check_style!
+          case style
+          when nil
+            raise Warning, "Auto Correction does not work for #{cop_name}. " \
```

Comment:
Auto-correction

On a related note - why should we check whether enforce style is set or not? And why should we do it in the auto-correction?